### DISCUSSION TITLE
What is the difference between executing 'valid' and 'test'?

### DISCUSSION THREAD

**Discussion by zte-yangjizhe** [Codium-ai/AlphaCodium#22] [Q&A] [answered]
What is the difference between executing 'valid' and 'test'?

**Comment by mrT23** [comment, marked as answer]
https://huggingface.co/datasets/deepmind/code_contests

<img width="811" alt="image" src="https://github.com/Codium-ai/AlphaCodium/assets/21198860/9b3caf05-e424-40e6-9122-89992844d5a5">

**Reply by silkflowers4** [reply]
I have absoloutely no idea why you guys decided to skip the training set entirely written the journal. The quality of codium depends on solving competitive coding problems accurately but if you skip the training and go straight into validation and test, you aren't feeding it enough data to become accurate. Can this be fixed? can this design implementation be explained?
